packages/runtime/runtime-utils/src/test/requestParser.spec.ts

```diff
@@ -47,20 +48,57 @@ describe("RequestParser", () => {
             requestParser = RequestParser.create({ url: "/dataStoreId//some/route/" });
         });
         it("Create request from part 0", () => {
-            assert.equal(requestParser.createSubRequest(0)?.url, "dataStoreId/some/route");
+            assert.strictEqual(requestParser.createSubRequest(0)?.url, "/dataStoreId/some/route");
         });
         it("Create request from part 1", () => {
-            assert.equal(requestParser.createSubRequest(1)?.url, "some/route");
+            assert.strictEqual(requestParser.createSubRequest(1)?.url, "/some/route");
         });
         it("Create request from part 2", () => {
-            assert.equal(requestParser.createSubRequest(2)?.url, "route");
+            assert.strictEqual(requestParser.createSubRequest(2)?.url, "/route");
         });
         it("Create request from parts length ", () => {
-            assert.equal(requestParser.createSubRequest(3)?.url, "");
+            assert.strictEqual(requestParser.createSubRequest(3)?.url, "/");
         });
         it("Create request from invalid part ", () => {
             assert.throws(() => requestParser.createSubRequest(4));
             assert.throws(() => requestParser.createSubRequest(-1));
         });
     });
+    describe(".createSubRequest with query params", () => {
```

Comment:
Any chance you can refactor this code (everything in describe() behind test(url) function, and call test() twice:
test("/dataStoreId/some/route/?queryparams");
test("dataStoreId/some/route/?queryparams");

all the it() methods should result in exactly same outcome!